### ISSUE TITLE
Double wordfile args

### ISSUE THREAD

**Issue by szepeviktor** [redacted/XKCD-password-generator#45]
[Here](https://github.com/redacted/XKCD-password-generator/commit/d495eefa1e8203ace70aa535d81702bbc7082453#diff-ea3c9f7c41da087db6bb248ddfb99f42R302) we have a section that **overwrites** `wordfile`.

Please see these two executions:
```bash
python3 /usr/local/bin/xkcdpass -V -w /usr/local/src/debian-server-tools/security/password2remember_hu.txt
The supplied word list is located at /usr/local/lib/python3.4/dist-packages/xkcdpass/static/default.txt.
Your word list contains 38271 words, or 2^15.22 words.
A 6 word password from this list will have roughly 91 (15.22 * 6) bits of entropy,
assuming truly random word selection.
reticule analogy unnerve Darold thereunto clinger
[root@szerver4:/usr/local/lib/python3.4/dist-packages/xkcdpass:21:03:21:568]

python3 /usr/local/bin/xkcdpass -V /usr/local/src/debian-server-tools/security/password2remember_hu.txt
The supplied word list is located at /usr/local/src/debian-server-tools/security/password2remember_hu.txt.
Your word list contains 9836 words, or 2^13.26 words.
A 6 word password from this list will have roughly 79 (13.26 * 6) bits of entropy,
assuming truly random word selection.
plusz tekint megnyitja kituno toebbiben irasom
```

So you may only specify wordfile without `-w`.

**Comment by redacted** [comment]
Thanks for reporting this. Fixed in master and v1.8.0 release to properly require `-w` or `--wordfile`.

**Comment by szepeviktor** [comment]
Thank you for fixing it.
